Cargo.toml

```diff
@@ -14,6 +14,13 @@ Timeouts for futures.
 
 [dependencies]
 
+[target.wasm32-unknown-unknown.dependencies]
+instant = { version = "0.1.2", features = ["wasm-bindgen"] }
+wasm-bindgen = "0.2.55"
+futures = "0.3.1"
+web-sys = "0.3.32"
+parking_lot = "0.10.0"
```

Comment:
Maybe we could do without `parking_lot` here? It seems it could reduce our compilation times, and in the browser it doesn't provide us with much speed improvement either way.

Reply:
Good idea